README.md

```diff
@@ -153,7 +153,7 @@ toggleActor.send({ type: 'TOGGLE' });
 // => logs 'active', { count: 1 }
 
 toggleActor.send({ type: 'TOGGLE' });
-// => logs 'inactive', { count: 1 }
+// => logs 'inactive', { count: 2 }
```

Comment:
This is not correct. This machine doesn't increment the counter when transitioning from the active state to the inactive state so the counter's value stays the same